rust/cubestore/cubestore/src/metastore/mod.rs

```diff
@@ -2848,7 +2848,8 @@ impl RocksMetaStore {
         }
 
         let aggregate_columns = table_id.get_row().aggregate_columns();
-        if aggregate_columns.is_none() {
+        if aggregate_columns.is_empty() {
+            //TODO check that index column is not in aggregate columns
```

Comment:
In the long run, it is harder to fix TODOs out of band than fixing them right now. It comes down to the cost of establishing the proper brain context: now you have uploaded all this stuff in your brain and ready to go, 3 months later the lane you (or someone else) might be a bit fuzzy on details, and would be harder for you to fix the TODO.

Barring that, consider "// TODO(waralex) Check ..."